testsuite/standalone/src/test/java/org/wildfly/core/test/standalone/mgmt/api/core/ModuleInfoTestCase.java

```diff
@@ -0,0 +1,173 @@
+                        "org/jboss/logmanager/formatters",
+                        "META-INF",
+                        "org/jboss/logmanager/filters",
+                        "org/jboss/logmanager/config",
+                        "META-INF/maven",
+                        "org/jboss/logmanager/handlers",
+                        "META-INF/maven/org.jboss.logmanager"
+                ]
+            },
+            {
+                "type" => "org.jboss.modules.NativeLibraryResourceLoader",
+                    "paths" => undefined
+            }
+            ]
+        }*/
+
+        // load module.xml
+        ModuleIdentifier identifier = ModuleIdentifier.fromString(TARGET_MODULE_NAME + ":main");
+        Module module = loadModule(identifier);
+
+        // run module-info operation
+        ModelNode op = Util.createEmptyOperation("module-info", RESOURCE);
+        op.get(NAME).set(TARGET_MODULE_NAME);
+        ModelNode response = getModelControllerClient().execute(op);
+        ModelNode result = validateResponse(response);
+
+        // compare module name
+        Assert.assertEquals("Unexpected name", TARGET_MODULE_NAME + ":main", result.get(NAME).asString());
+
+        // compare dependencies
+        List<ModelNode> dependencies = result.get("dependencies").asList();
+        DependencySpec[] xmlDependencies = module.getDependencies();
+        Map<String, ModuleDependencySpec> moduleDependencies = new HashMap<>();
+        for (DependencySpec d : xmlDependencies) {
+            if (d instanceof ModuleDependencySpec) {
+                ModuleDependencySpec mds = (ModuleDependencySpec) d;
+                moduleDependencies.put(mds.getIdentifier().getName() + ":" + mds.getIdentifier().getSlot(), mds);
+            }
+        }
+        int foundDependencies = 0;
+        for (ModelNode d : dependencies) {
+            String dependencyName = d.get("module-name").asString();
+            Assert.assertNotNull("Expected dependency", moduleDependencies.get(dependencyName));
+            foundDependencies++;
+        }
+        Assert.assertEquals("# of dependencies in module.xml != # of dependencies in module-info output",
+                moduleDependencies.size(), foundDependencies);
+
+        // compare local paths
+        List<ModelNode> resourceLoaders = result.get("resource-loaders").asList();
+        // there should be only one jar loader in org.jboss.logmanager module
+        List<String> paths = new ArrayList<>();
+        for (ModelNode n : resourceLoaders) {
+            if (n.get("type").asString().equals("org.jboss.modules.JarFileResourceLoader")) {
+                for (ModelNode node : n.get("paths").asList()) {
+                    paths.add(node.asString());
+                }
+                break;
+            }
+        }
+        String xmlLocalPaths = module.getClassLoader().getLocalPaths().toString();
+        Assert.assertEquals(xmlLocalPaths, paths.toString());
```

Comment:
This should be `Assert.assertTrue(module.getClassLoader().getLocalPaths().containsAll(paths));`.